doc/searches.rst

```diff
@@ -41,24 +49,26 @@ there names don't clash. Use a descriptive name like: 'customer_invoices' and 'c
 
 .. tip::
 
-    We can also use the FieldSetBuilder to import the fields from models
-    using the :doc:`metadata` component.
+    You can also use the FieldSetBuilder to import the fields from models
```

Comment:
before i forget, you should have a paragraph convincing me or your readers first how this compares to the regular approach, how is it that this library is a replacement or solves a problem that otherwise it is not solved by any other approach, or if it is better then in which way it is and how it suites the roadmap of an application.